### PR TITLE
Support reading swarm version from plugins.yaml

### PR DESCRIPTION
**Pull request by nre-ableton**
- Support reading swarm version from plugins.yaml
- Version 0.1.0

### CODE REVIEW

File: tasks/get-swarm-client-version.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- name: Read plugins YAML file
+  delegate_to: localhost
+  ansible.builtin.slurp:
+    src: "{{ jenkins_plugins_file }}"
+  register: jenkins_plugins_file_contents
+
+- name: Parse plugins YAML file to a variable
+  ansible.builtin.set_fact:
+    jenkins_plugins_contents: >-
+      {{ jenkins_plugins_file_contents.content | b64decode | from_yaml }}
```
Comment (ala-ableton) [inline tasks/get-swarm-client-version.yml:11]: What about
```yaml
- name: Parse plugins YAML file to a variable
  ansible.builtin.set_fact:
    jenkins_plugins_contents: >-
      {{ lookup('ansible.builtin.file', jenkins_plugins_file) | from_yaml }}
```
?

File: tasks/get-swarm-client-version.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- name: Set jenkins_swarm_client_version fact
+  ansible.builtin.set_fact:
+    jenkins_swarm_client_version: "{{ plugin_version_item['source']['version'] }}"
+  loop: "{{ jenkins_plugins_contents['plugins'] }}"
+  loop_control:
+    loop_var: plugin_version_item
+  when: plugin_version_item['artifactId'] == 'swarm'
```
[the remark this answers is not in the document]
Comment (nre-ableton) [reply tasks/get-swarm-client-version.yml:19]: Yeah, usually it causes problems for me by making loops terminate early, but in this case it's very convenient. :slightly_smiling_face:

File: tasks/get-swarm-client-version.yml (hunk 1 above)
Comment (nre-ableton) [reply tasks/get-swarm-client-version.yml]: Ooooh, very nice! Fixed in d88ce6b.

**Review by ala-ableton — APPROVED** [review]

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**